src/vars/images.yml

```
@@ -1,11 +1,11 @@
 candlepin_container_image: quay.io/foreman/candlepin
-candlepin_container_tag: "4.4.14"
+candlepin_container_tag: "foreman-nightly"
```

I'd love to have 1 variable to update for every release. Something like:
```suggestion
candlepin_container_tag: "foreman-{{ foreman_container_tag }}"
```

Perhaps introduce something like `container_tag_stream: nightly` and then consistently use that for all the containers? That allows changing a single container easily while also making branching easier.